modules/tile-converter/src/i3s-converter/helpers/load-3d-tiles.ts

```diff
@@ -62,7 +67,53 @@ export const loadTile3DContent = async (
       assetGltfUpAxis: (sourceTileset.asset && sourceTileset.asset.gltfUpAxis) || 'Y'
     }
   };
-  const tileContent = await load(sourceTile.contentUrl, sourceTileset.loader, loadOptions);
+  const tileContent = await loadWithOptions(
+    sourceTile.contentUrl,
+    sourceTileset.loader,
+    loadOptions
+  );
 
   return tileContent;
 };
+
+/**
+ * Load a resrource with load options and .3tz format support
+ * @param url - resource URL
+ * @param loader - loader to parse data (Tiles3DLoader / CesiumIonLoader)
+ * @param loadOptions - 3d-tiles loader options
+ * @returns 3d-tiles resource
+ */
+export async function loadWithOptions(
```

Comment:
We normally just call this `load`, all load call have options. Maybe update name to indicate what we are loading instead, or what we are loading from?

Reply:
Done